Litres.js

```diff
@@ -0,0 +1,175 @@
+{
+	"translatorID": "02bde528-c86d-42d5-904b-e74f85bd45f9",
+	"label": "Litres",
+	"creator": "Ilya Zonov",
+	"target": "^https://(www\\.)?litres\\.ru/",
+	"minVersion": "5.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcsibv",
+	"lastUpdated": "2025-03-29 17:26:32"
+}
+
+/*
+	***** BEGIN LICENSE BLOCK *****
+
+	Copyright © 2025 Ilya Zonov
+
+	This file is part of Zotero.
+
+	Zotero is free software: you can redistribute it and/or modify
+	it under the terms of the GNU Affero General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	Zotero is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+	GNU Affero General Public License for more details.
+
+	You should have received a copy of the GNU Affero General Public License
+	along with Zotero. If not, see <http://www.gnu.org/licenses/>.
+
+	***** END LICENSE BLOCK *****
+*/
+
+
+function detectWeb(doc, url) {
+	if (url.includes('/book/')) {
+		return 'book';
+	}
+	return false;
+}
+
+async function doWeb(doc, url) {
+	await scrape(doc, url);
+}
+
+async function scrape(doc, url = doc.location.href) {
+	const translator = Zotero.loadTranslator('web');
+
+	// Embedded Metadata
+	translator.setTranslator('951c027d-74ac-47d4-a107-9c3069ab7b48');
+	translator.setDocument(doc);
+
+	translator.setHandler('itemDone', function (obj, item) {
+		item.publicationTitle = "";
+
+		item.title = text(doc, 'h1[itemprop="name"]');
+
+		const authors = doc.querySelectorAll('div[data-testid="art__author--details"] a[data-testid="art__personName--link"] span');
+		for (const author of authors) {
+			item.creators.push(ZU.cleanAuthor(author.textContent, 'author'));
+		}
+
+		item.series = text(doc, 'div[data-testid="art__inSeries--title"] a').replaceAll(/[«»]/g, '');
+
+		const seriesInfo = text(doc, 'div[data-testid="art__inSeries--title"] div');
+		const seriesMatch = seriesInfo.match(/([0-9]+) .+ из [0-9]+ в серии/);
+		if (seriesMatch) {
+			item.seriesNumber = seriesMatch[1];
+		}
+
+		item.ISBN = text(doc, 'span[itemprop="isbn"]');
+
+		item.publisher = text(doc, 'div[data-testid="book__characteristicsCopyrightHolder"] a');
+
+		item.complete();
+	});
+
+	const em = await translator.getTranslatorObject();
+	em.itemType = 'book';
+	em.doWeb(doc, url);
+}
+
+/** BEGIN TEST CASES **/
+var testCases = [
+	{
+		"type": "web",
+		"url": "https://www.litres.ru/book/daron-asemoglu/pochemu-odni-strany-bogatye-a-drugie-bednye-proishozhdenie-10748980/",
+		"items": [
+			{
+				"itemType": "book",
+				"title": "Почему одни страны богатые, а другие бедные. Происхождение власти, процветания и нищеты",
+				"creators": [],
```

Comment:
Looks like we're missing creators.

Reply:
Fixed. Also I have added multiple support.